.talismanrc

```diff
@@ -13,7 +13,20 @@ fileignoreconfig:
     checksum: ca23dfbb1e6b7ba8814a1a19075949c9a01eee92fa9b8a31d0f9016dd33d79ba
   - filename: "*.woff*"
     checksum: 06af891e663221f6014dff8f7b49cfa0a578680f44ba1ec2dd3b44bd8d3b1f05
-  - filename: src/test/resources/application.properties
-    checksum: 9168889b085ab87ed30b13ed6adb4f2b197c1dec81f8fa24c5eaa0001daa7ac7
   - filename: doc/adr/0006-url-format-in-qr-code.md
     checksum: d03d40784b228768e836c22df9dc3925048e42c3c0e40dc7ad9ef91804ab0dce
+  - filename: src/test/resources/application.yaml
+    checksum: d0db4798a099371f3af9e6119dcf407e441590e6b3d76bc805e5607119b7125f
+  - filename: src/main/kotlin/de/bund/digitalservice/useid/config/ApiProperties.kt
+    checksum: 41d614a71b4854ed5832eaf804712a1252c9bc8406616137303b35fcdec79c82
+  - filename: src/main/kotlin/de/bund/digitalservice/useid/apikeys/ApiKeyAuthenticationManager.kt
+    checksum: c784f489b09de0e3de46ca9e8f6d52521f557542b3b37c82032b13c3c9c4cbff
+  - filename: src/main/kotlin/de/bund/digitalservice/useid/apikeys/ApiKeyAuthenticationConverter.kt
+    checksum: 2881fab3f14e061c4a3327046e5376a1b5f1841cd57c67edc6aa05dd07f84b82
+  - filename: src/test/kotlin/de/bund/digitalservice/useid/apikeys/ApiKeyAuthenticationManagerTest.kt
+    checksum: d5b224b9b4f8f32193dc8d997e5b2a2c24df79cbbc115ff73701c5676ff7cf63
+  - filename: src/test/kotlin/de/bund/digitalservice/useid/apikeys/ApiKeyAuthenticationConverterTest.kt
+    checksum: 7acd129e6ce59738f1b819adc2b482b49a43ffdbe1e21a33bbb33443afaed88d
+  - filename: src/main/kotlin/de/bund/digitalservice/useid/apikeys/ApiKeyAuthenticationToken.kt
+    checksum: 61c0baa54d9ef1e1281e19db74dcb8be96c37be2b09c41e4d8d9bdca8865f69c
```

Comment:
is it possible to use glob in here? it's similar to the fonts above with asterisk `"*.woff*"`
so could we try to use `src/main/kotlin/de/bund/digitalservice/useid/apikeys/*` instead?

Reply:
Good hint working with wildcards. Makes the file much better readable.